devito/core/gpu_openmp.py

```diff
@@ -2,18 +2,26 @@
 
 import cgen as c
 
+from copy import deepcopy
 from devito.core.operator import OperatorCore
 from devito.data import FULL
 from devito.exceptions import InvalidOperator
-from devito.ir.iet import Callable, ElementalFunction, MapExprStmts
+from devito.ir.equations import DummyEq
+from devito.ir.iet import (Block, Call, Callable, ElementalFunction, FindNodes,
+                           LocalExpression, List, MapExprStmts, Transformer)
 from devito.logger import warning
-from devito.mpi.routines import CopyBuffer, SendRecv, HaloUpdate
+from devito.mpi.distributed import MPICommObject
+from devito.mpi.routines import (CopyBuffer, HaloUpdate, IrecvCall, IsendCall, SendRecv,
+                                 MPICallable)
 from devito.passes.clusters import (Lift, cire, cse, eliminate_arrays, extract_increments,
                                     factorize, fuse, optimize_pows)
 from devito.passes.iet import (DataManager, Storage, Ompizer, OpenMPIteration,
                                ParallelTree, optimize_halospots, mpiize, hoist_prodders,
                                iet_pass)
+from devito.symbolics import Byref
 from devito.tools import as_tuple, filter_sorted, timed_pass
+from devito.types import Symbol
+from sympy import Function
```

Comment:
this import should be moved under `import cgen as c` as it's not a devito import, but rather an import from a devito dependency